<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aiyush Gupta - HEX, DEN, BIN Converter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="title_label">
       <property name="font">
        <font>
         <pointsize>22</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Hexadecimal Denary Binary Converter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="input_groupBox">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QPushButton" name="convert_pushButton">
          <property name="text">
           <string>Convert</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Type</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <widget class="QRadioButton" name="denaryType_radioButton">
             <property name="text">
              <string>Denary</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="hexadecimalType_radioButton">
             <property name="text">
              <string>Hexadecimal</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QRadioButton" name="binaryType_radioButton">
             <property name="text">
              <string>Binary</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="autoType_radioButton">
             <property name="text">
              <string>Auto</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="input_lineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabs">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="calculationTab">
        <attribute name="title">
         <string>Calculation</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTextEdit" name="calculation"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="consoleTab">
        <attribute name="title">
         <string>Console</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QTextEdit" name="console"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="creditsTab">
        <attribute name="title">
         <string>Credits</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QTextEdit" name="credits"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
